packages/mui-base/package.json

```diff
@@ -60,6 +60,7 @@
     "react-is": "^17.0.2"
   },
   "devDependencies": {
+    "@mui/material": "^5.2.7",
```

Comment:
This looks strange to me because mui-base should not use anything from the design system package (mui-material). cc @michaldudak

Reply:
The package itself doesn't use it, only some of its tests do. I can imagine there to be legitimate use cases for that.